### ISSUE TITLE
Remove @FrameworkAgnostic Markers

### ISSUE THREAD

**Issue by killianlevacher** [Trusted-AI/adversarial-robustness-toolbox#732]
**Is your feature request related to a problem? Please describe.**
Once issue #729 is complete, @frameworkagnostic markers (which were needed when using Travis) should be removed since we not not anymore reach any time limit for tests. This will guarantee that each test is run by each framework.

**Comment by beat-buesser** [comment]
Duplicate of #807
